### PR TITLE
Rubric create test - group(s)

### PR DESCRIPTION
**Pull request by MitchelPaulin**

### CODE REVIEW

File: test/d2l-rubric-criteria-groups-editor.test.js [hunk 1]
```diff
@@ -1,76 +1,111 @@
...
+		before(async() => {
 
-			expect(element).to.exist;
-			expect(element.localName).to.equal('d2l-rubric-criteria-groups-editor');
+			multipleGroupElement = await fixture(html`<d2l-rubric-criteria-groups-editor
+				href="../data/instructor/rubrics/organizations/1/8/groups.json"
+				token="test"
+			></d2l-rubric-criteria-groups-editor>`);
+
+			await new Promise((resolve) => { setTimeout(resolve, 1000); });
```
Comment (MitchelPaulin) [inline test/d2l-rubric-criteria-groups-editor.test.js:51]: Open to alternatives to this here, the href I am using has a lot of data and takes a while to load. Unfortunately `fixtureWaitForChangeEvent` doesn't work. I was looking at `fixtureWaitForSpecificChangeEvent` but couldn't think of a helpful event.

File: test/components/static-data/rubrics/organizations/text-only/199/groups/176.json [hunk 2]
```diff
@@ -9,8 +9,18 @@
         {
```
Comment (devpow112) [inline test/components/static-data/rubrics/organizations/text-only/199/groups/176.json:9]: Are you able to switch away from `test/components/static-data` or is there no nice equivalent in `/data`? If it's too much work that's fine. I'd just like to eventually kill all the data we didn't auto generate (anything not under `/data`)

File: test/d2l-rubric-criterion-editor.test.js [hunk 3]
```diff
@@ -3,15 +3,15 @@ import './components/static-data/rubrics/organizations/text-only/199/groups/176/
...
+import sinon from 'sinon';
 
 const eventOptions = { bubbles: true, cancelable: false, composed: true };
 
 describe('d2l-rubric-criterion-editor', () => {
 	let sandbox;
 
 	before(() => {
-		sandbox = createSandbox();
+		sandbox = sinon.createSandbox();
```
Comment (devpow112) [inline test/d2l-rubric-criterion-editor.test.js:14]: Why the change to use `sinon.createSandbox`? Just curious if there was a technical reason.

Comment (MitchelPaulin) [reply test/d2l-rubric-criterion-editor.test.js:14]: I think thats left over from me trying to debug some failing tests, I can switch it back

File: test/d2l-rubric-criteria-group-editor-create.test.js [hunk 4]
```diff
@@ -10,12 +10,16 @@ describe('d2l-rubric-criteria-group-editors', () => {
 	const performActionStub = sandbox.stub(RubricCriteriaGroupEditor.prototype, 'performSirenAction').callsFake(() => Promise.resolve());
 
 	before(async() => {
-		element = await fixtureWaitForChangeEvent(
-			html`<d2l-rubric-criteria-group-editor show-group-name></d2l-rubric-criteria-group-editor>`,
-			'static-data/rubrics/organizations/text-only/199/groups/176.json'
+		element = await fixture(
+			html`<d2l-rubric-criteria-group-editor
+				href="../data/instructor/rubrics/organizations/1/2/groups/18.json"
```
Comment (devpow112) [inline test/d2l-rubric-criteria-group-editor-create.test.js:15]: I thin base on how the test runner is configured this can just be `/data/instructor/` instead of `../data/instructor/`. I may be wrong though.

Comment (imcdonald-d2l) [reply test/d2l-rubric-criteria-group-editor-create.test.js]: Yeah, should just be `/` not `../`

File: test/utilities.js [hunk 5]
```diff
@@ -77,3 +77,19 @@ export const isVisible = (element) => {
...
+export const isRejected = async(promise) => {
+	try {
+		await promise;
+	} catch (_) {
+		return true;
+	}
+	return false;
+};
+
+export const waitOneSecond = async() => {
```
Comment (MitchelPaulin) [inline test/utilities.js:93]: Added this as a hack until we can figure out something better

Comment (imcdonald-d2l) [inline test/utilities.js:93]: Could we make this a utility that takes in a `ms` parameter so we can control the timeout? Would be more useful in the future if we need to tweak the duration.

Comment (imcdonald-d2l) [reply test/utilities.js:93]: When I was playing around with accessibility tests I did something very similar. Don't know of any more graceful solutions to the problem this is trying to solve at the moment.

File: test/d2l-rubric-criterion-editor.test.js [hunk 6]
```diff
@@ -233,6 +233,11 @@ describe('d2l-rubric-criterion-editor', () => {
 				const subElement = element.shadowRoot.querySelector('#remove');
 				expect(subElement).to.be.null;
 			});
+
+			it('dropdown is hidden', () => {
+				const subElement = element.shadowRoot.querySelector('d2l-dropdown-more');
+				expect(subElement).to.be.null;
```
Comment (imcdonald-d2l) [inline test/d2l-rubric-criterion-editor.test.js:239]: Would `to.not.exist` be better than `to.be.null`?

Comment (MitchelPaulin) [reply test/d2l-rubric-criterion-editor.test.js:239]: querySelector doesn't return undefined, so checking for null directly here is fine

Comment (devpow112) [reply test/d2l-rubric-criterion-editor.test.js:239]: Interesting. Good to know. Wonder if we are over checking anywhere (checking `null` and `undefined`) in our code when we have a return from `querySelector`

**Review by devpow112 — APPROVED** [review]
Looks good to me :tada:
